openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/IBMBeanExtension.java

```diff
@@ -1,42 +1,42 @@
-* Licenses when the conditions for such availability set forth in the 
-* Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
-* version 2 with the GNU Classpath Exception [1] and GNU General Public License,
-* version 2 with the OpenJDK Assembly Exception [2].
-* 
-* [1] https://www.gnu.org/software/classpath/license.html
-* [2] http://openjdk.java.net/legal/assembly-exception.html
-*
-* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
-*******************************************************************************/
-
-package net.openj9.test.extensions;
-
-import java.io.PrintWriter;
-
-import javax.management.MBeanServerConnection;
-
-public abstract class IBMBeanExtension {
-	protected PrintWriter out = null;
-	protected MBeanServerConnection mbs = null;
-	protected boolean isLocal = false;
-	protected boolean isProxy = false; 
-	
-	public IBMBeanExtension(PrintWriter out, MBeanServerConnection mbs, boolean isLocal, boolean isProxy) {
-		this.out = out;
-		this.mbs = mbs;
-		this.isLocal = isLocal;
-		this.isProxy = isProxy;
-	}
-	
-	public abstract void execute(); 
-}
+/*******************************************************************************
+* Copyright (c) 2017, 2019 IBM Corp. and others
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License 2.0 which accompanies this distribution
+* and is available at http://eclipse.org/legal/epl-2.0 or the Apache License, 
+* Version 2.0 which accompanies this distribution and is available at 
+* https://www.apache.org/licenses/LICENSE-2.0.
+* 
+* This Source Code may also be made available under the following Secondary
+* Licenses when the conditions for such availability set forth in the 
+* Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+* version 2 with the GNU Classpath Exception [1] and GNU General Public License,
+* version 2 with the OpenJDK Assembly Exception [2].
+* 
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+*******************************************************************************/
+
+package net.openj9.test.extensions;
+
+import java.io.PrintWriter;
+
+import javax.management.MBeanServerConnection;
+
+public abstract class IBMBeanExtension {
+	protected PrintWriter out = null;
+	protected MBeanServerConnection mbs = null;
+	protected boolean isLocal = false;
+	protected boolean isProxy = false; 
+	
+	public IBMBeanExtension(PrintWriter out, MBeanServerConnection mbs, boolean isLocal, boolean isProxy) {
+		this.out = out;
+		this.mbs = mbs;
+		this.isLocal = isLocal;
+		this.isProxy = isProxy;
+	}
+	
+	public abstract void execute(); 
+}
```

Comment:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.